judoscale-ruby/test/request_middleware_test.rb

```diff
@@ -35,9 +35,11 @@ def call(env)
           Judoscale.configure { |config| config.api_base_url = "http://example.com" }
         }
 
-        it "passes the request up the middleware stack" do
-          middleware.call(env)
-          _(app.env).must_equal(env)
+        it "passes the request env up the middleware stack, returning the app's response" do
+          response = middleware.call(env)
+
+          _(response).must_equal app
+          _(app.env).must_equal env
```

Comment:
Expanding tests a bit to make sure the response is actually returned from the middleware always.